packages/medusa/src/api-v2/store/payments/payment-sessions/route.ts

```diff
@@ -0,0 +1,84 @@
+import {
```

Comment:
thought(non-blocking): Not convinced that this mismatch between endpoint path and response is the right approach. It would break with the general convention of our Rest API.

Reply:
@olivermrbl I kept it as it was originally, but I completely agree that we shouldn't have this mismatch. I'll look into changing it.

On a more general note what do you think our payment-related APIs should look like?

I added them under `payments`, but honestly I am not convinced if that's the best way?